pkgs/applications/audio/aacgain/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv, fetchgit, unzip, m4, libtool, automake, autoconf, mp3gain, mp4v2, faad2 }:
+
+stdenv.mkDerivation {
+  name = "aacgain-1.9";
+  srcs = [
+    (fetchgit {
```

Comment:
Please use fetchFromGitHub, with the release tag as `rev`, that avoids cloning the full repository.